Description: Convert documentation text from GB2312 to UTF-8
 Committed to zh-autoconvert (0.3.16-4) on 2015-11-17.
Author: Anthony Fok <foka@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2015-11-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,13 +1,13 @@
-һЩgccչл law at redhat.comĲ
+修正一些依赖gcc扩展的特征，感谢 law at redhat.com的补丁程序。
 					Yu Guanghui <ygh@debian.org>
 					2004.03.22
 
-չb2g table, лHsieh Yu-T'ing <victor at cirx.org.anti-spam>Ĳ.
+扩展b2g table, 感谢Hsieh Yu-T'ing <victor at cirx.org.anti-spam>的补丁程序.
 Autoconvert 0.3.14 Release.
 					Yu Guanghui <ygh@debian.org>
 					2003.12.12
 __
-Licenseļ, hzttyİȨ.
+更改License文件, 加入hztty代码的版权声明.
 Autoconvert 0.3.13 Release.
 
 					Yu Guanghui <ygh@debian.org>
@@ -26,87 +26,87 @@
 					Yu Guanghui <ygh@debian.org>
 					2000.01.08
 __
-big5uniתģ,ֹת.
-					ڹ
+big5的uni转换模块修正,现在手工转换可以正常工作了.
+					于广辉
 					Yu guanghui <ygh@debian.org>
 					2000.11.28
 __
-µת, ͨ㷨Զ˵תģ.
-big5uniģ,Ҫһ޸.
-					ڹ
+基本完成新的转换框架, 通过广度优先算法自动组合适宜的转换模块.
+但是big5的uni模块有问题,需要进一步修改.
+					于广辉
 					Yu Guanghui <ygh@debian.org>
 					2000.11.28
 --
-һxchatgb/big5Զתplugins
-					ڹ
+加入一个xchat的gb/big5自动转码plugins。
+					于广辉
 					Yu Guanghui <ygh@debian.org>
 					2000.11.25
 __
-ַжϺ,Ha Shao<hashao@china.com>ṩ
-ַָ
+更新字符判断函数,Ha Shao<hashao@china.com>提供。
+增加指定输入字符功能
 
-					  ڹ
+					  于广辉
 					  Yu Guanghui <ygh@debian.org>
 					  2000.8.31
 __
-һʼMIMEתű,ԴMIMEʼ͸.
-лAnthory Fok<foka@debian.org>ṩԭʼű.
+加入一个邮件MIME转换脚本,可以处理MIME编码的邮件主题和附件.
+感谢Anthory Fok<foka@debian.org>提供原始脚本.
 
-					     ڹ
+					     于广辉
 					     Yu Guanghui<ygh@dlut.edu.cn>
 					     2000.6.14
 __
 only split hzconvert to libhz
-					     ڹ
+					     于广辉
                                              Yu Guanghui<ygh@dlut.edu.cn>
                                              2000.1.5
 __
-˵8192ֽʱְֵܳbug.
-ʹhzttyhz2gbԭĵĺȷ8192ֽڵ
-autogb,autob5ϲһ,ó
-					      ڹ
+改正了当输入大于8192字节时可能出现半个汉字的bug.
+重新使用hztty的hz2gb函数，原来的的函数不能正确处理大于8192字节的情况。
+把autogb,autob5合并到一个程序里面,用程序名区别。
+					      于广辉
 					      Yu Guanghui<ygh@dlut.edu.cn>
 					      1999.12.28
 __
-ĽGB/Big5ʶ𷽷,ڸƵͳƵĻ˻ڶԺ
-ֱͳƵıʶ𷽷.
-					      ڹ
+改进了GB/Big5的智能识别方法,在高频词统计的基础上增加了基于对汉字
+编码直接统计的备用识别方法.
+					      于广辉
 					      Yu Guanghui<ygh@dlut.edu.cn>
 					      1999.11.20
 __
-޸hz2gbһbug.
-hzgb/big5ת
+修复hz2gb函数中一个bug.
+hz到gb/big5转换正常了
 						
-					       ڹ
+					       于广辉
 					       Yu Guanghui<ygh@dlut.edu.cn>
 					       1999.10.21  
 --
-ֲ hztty-2.0Ĳִ,ֱӽк,Ҫⲿת.
-д hzgbתֵĴ,ĳɺ.
-ûhzttyhz2gb,hzttyhztogb̫,ûбҪ.
+移植了 hztty-2.0的部分代码,直接进行函数调用,不需要外部转换程序了.
+重写了 hz到gb转换部分的代码,改成函数形势.
+没有用hztty的hz2gb程序,hztty的hztogb太繁琐,没有必要.
 
-						ڹ
+						于广辉
                                                 Yu Guanghui<ygh@dlut.edu.cn>
 						1999.10.20
 --
 
-Ϊutf-convertİȨ,ڰＯhcgb/big5ת.
-޸һɵжʱ(base64)޷bug.
+因为utf-convert的版权问题,所以在包里集成了hc用来做gb/big5编码转换.
+修复一个造成当有多个附件时(base64)无法解码的bug.
 
 						
-                                                ڹ
+                                                于广辉
                                                 Yu Guanghui<ygh@dlut.edu.cn>
 						1999.10.14
 --
-ǰһdebian chineseҶʹGB,BIG5,HZ˺óһʱ,Ͳ
-дԶhz,gb,chineseת뷨.дıȽϴæ,
-ֻǿԹ,϶вbug.ֻ³ȱȽϳʱ
-ȷ.ٹһʱٸĽ,ʵûʱ.
+前一阵debian chinese大家对使用GB,BIG5,HZ讨论了好长一段时间,就产生了
+写个程序自动完成hz,gb,chinese的转换的想法.不过这个程序写的比较匆忙,
+只是可以工作而以,肯定有不少bug的.而且只有在文章长度比较长的时候才能
+正确处理.等再过一段时间再改进吧,现在实在没时间了.
 
-Ҫutf-converter packages
+需要utf-converter packages
 
 
-						ڹ
+						于广辉
 						Yu Guanghui<ygh@dlut.edu.cn>
 						1999.5.29
 
--- a/LICENSE
+++ b/LICENSE
@@ -1,6 +1,6 @@
 /*
 AutoConvert, a Chinese HZ/GB/Big5 encodings auto-converter
-Copyright (C) 1999  ڹ  Yu Guanghui <ygh@debian.org>
+Copyright (C) 1999  于广辉  Yu Guanghui <ygh@debian.org>
 
 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
--- a/Readme
+++ b/Readme
@@ -1,27 +1,27 @@
 
 			AutoConvert
 
-    AutoConvertһֱԶʶתĳ,ĿǰԶʶGB/Big5/HZ
-    ,תgb/big5/hz/uni/utf7/utf8.ı1000ʱ,GB/Big5
-    ʶ׼ȷʽӽ100%.HZʶҲ100%.
+    AutoConvert是一个汉字编码自动识别转换的程序,目前他可以自动识别GB/Big5/HZ
+    编码,可以转换gb/big5/hz/uni/utf7/utf8编码.当文本超过1000字时,对GB/Big5的
+    识别准确率接近100%.HZ识别率也近似100%.
 
-    Ŀ,Ҳ;procmailԵʼк
-    Զת.
+    这个程序最初的目的,也是它现在最大用途就是配合procmail对电子邮件进行汉字
+    编码自动转换.
 
-    AutoConvertڻܼ,ܴGB/Big5/HZϱ.
+    AutoConvert现在还很简单,不能处理GB/Big5/HZ混合编码.
 
-    AutoConvert÷ܼ,autoconvertĿ¼,makeͿ.
-    make install԰ autogb,autob5  $HOME/binĿ¼.
-    procmail.exampleprocmailͿ.
+    AutoConvert的用法很简单,在autoconvert目录下,make就可以了.
+    make install可以把 autogb,autob5 拷贝到 $HOME/bin目录.
+    按procmail.example设置procmail就可以了.
 
-    autogb,autob5ֱӴstdin,stdout, Ӳ, Զж
-    ,Ȼת.Ҳָͨ롢롣
+    autogb,autob5都直接从stdin读,输出到stdout, 如果不加参数, 将自动判断输入
+    编码,然后进行转换.也可以通过命令行指定输入、输出编码。
 
 
     URL: 
     banyan.dlut.edu.cn/~ygh
     ftp.dlut.edu.cn/pub/PEOPLE/saka/
 
-    							ڹ<ygh@debian.org>
+    							于广辉<ygh@debian.org>
 
 							2000.11.29
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Զunicodeж.
+尝试对unicode编码进行判断.
--- a/doc/API
+++ b/doc/API
@@ -1,9 +1,9 @@
 FIX ME: Add more example
 
-ͨlibhzṩӿںlibhz⡣
-1j_codeԱж
-2hz_setupĿǰڲתģ·Լⲿģ顣
-3hz_searchתģ飬ʼתģ
-4ҲԵhz_init³ʼģ
-5hz_convertִת
+建议通过libhz提供的三个接口函数调用libhz库。
+1、j_code对编码进行判断
+2、hz_setup目前用来把内部转换模块加入搜索路径，将来可以加入外部模块。
+3、hz_search组合转换模块，初始化转换模块
+4、也可以调用hz_init重新初始化模快
+5、hz_convert执行转换
 
--- a/doc/libhz.txt
+++ b/doc/libhz.txt
@@ -1,50 +1,50 @@
-		libhz ʹü
+		libhz 使用简介
 		
-libhzṩgb/big5/hzԶʶgb,big5,hz,utf8,utf7,unicode֮ת
-У
-include/hz.h   /*ͷļ*/
-lib/libhz.a    /*̬*/
-lib/libhz.so   /**/
+libhz提供了gb/big5/hz自动识别和gb,big5,hz,utf8,utf7,unicode之间的转换函数。
+其中：
+include/hz.h   /*头文件*/
+lib/libhz.a    /*静态库*/
+lib/libhz.so   /*共享库*/
 
-Example: gb/big/hzʶ
+Example: gb/big/hz识别
 
-incode=j_code(pbuf,count);벻Ҫֱʹj_code3.
-pbufҪǱַcountҪʶַ
-ֵhz.h涨塣
+incode=j_code(pbuf,count);请不要直接使用j_code3等.
+pbuf是要是别的字符串，count是要识别的字符数。
+返回值在hz.h里面定义。
 
 
-Example gb->big5ת
+Example： gb->big5转换
 
-һ: ֱӵת
+方法一: 直接调用转换函数
 ...
-gb2big_init();   /*ʼ*/
+gb2big_init();   /*初始化*/
 
-...		/*ѭ,pbuf,Ȼת*/
+...		/*循环,读到pbuf里,然后转换*/
 gb2big(pbuf,&count,0) 
-...		/*ѭ*/
+...		/*循环结束*/
 
- pbufָҪתַָ룬pbufǰһַĿռ,countҪת
-ַ0ָʹõڼbuffer.
+其中 pbuf是指向要转换的字符串的指针，pbuf前保留一个字符的空间,count是要转换的
+字符数，0是指定使用第几个buffer.
 
-ʹbufferԭΪ˷ֹpbufְְ֣ʱ־ͻ
-buffer汣棬´εgb2bigʱ־ͻᱻӵǰ档ע⣬16buffer
+使用buffer的原因是为了防止pbuf的最后出现半个字，当出现半个字时，这半个字就会在
+buffer里面保存，下次调用gb2big时，这半个字就会被添加到前面。注意，可以有16个buffer
 (0-15).
-ʹ÷Ҳơ
+其他函数使用方法也类似。
 
-: ʹhz_search()hz_convert()
-Ƽʹַ, searchʵת, convertת.
+方法二: 使用hz_search()和hz_convert()组合
+推荐使用这种方法, search可以组合最合适的转换函数, convert完成转换过程.
 
 hz_setup();
-ʼģݿ, ⲿģ.
+初始化模块数据库, 将来会允许调用外部模块.
 ...
 dist=search(incode,outcode,8);
-distҪϵתĸ, 0޷ת. 8ʾ8
-ת.
+dist是需要组合的转换函数的个数, 如果等于0表明无法完成转换. 8表示最多组合8个
+转换函数.
 ...
 ps=convert(pbuf,&count,0);
-ת, convert˳Ϻõĺ.
-pbuf봮, countΪתַ, 0ʾʹڲĴ浥Ԫ.
-ַconvertд.
+完成转换过程, convert可以自行顺序调用组合好的函数.
+pbuf是输入串, count为转换字符数, 0表示使用内部寄存单元号码.
+半个字符问题convert可以自行处理.
 
 
 						Yu Guanghui
